tests/integration/test_rex.py

```diff
@@ -1054,3 +1057,66 @@ def test_totimestamp(c, gpu):
         }
     )
     assert_eq(df, expected_df, check_dtype=False)
+
+
+@pytest.mark.parametrize(
+    "gpu",
+    [
+        False,
+        pytest.param(
+            True,
+            marks=(
+                pytest.mark.gpu,
+                pytest.mark.skipif(
+                    not DASK_CUDF_TODATETIME_SUPPORT,
+                    reason="Requires https://github.com/dask/dask/pull/9881",
```

Comment:
In line with the previous change, we can also expect a runtime error here instead of skipping the test for this case. 
I think `pytest.mark.xfail` with the `raises` param should work https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-xfail